test/common/libraries/os_properties.rb

```diff
@@ -27,6 +27,10 @@ def centos?
     inspec.os.name == 'centos'
   end
 
+  def centos_min_version?(version)
+    centos? && inspec.os.release.to_f >= version
+  end
```

Comment:
I am conflicted about this. We should know which version we run the test on, right?

Reply:
Also, I am not sure we need a library function. It looks very specific.

Reply:
> We should know which version we run the test on, right?

We do, but we could change it and the test code must work (at least for the version in which we have the information on what to test)

> Also, I am not sure we need a library function. It looks very specific.

I can agree on that, moving it to the test code